index.rst

```diff
@@ -231,78 +279,148 @@ are captured correctly.
 
 .. _Control Utilities:
 
-Control Utilities
------------------
+Control Package Utilities
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Control utilities are analogous to the utilities discussed in `Observing Utilities`_, but have been evolved and moved
-into production code. Sanctioned Control Utilities will exist at multiple levels. These utilities will primarily be
-called by jobs for the Queue, but not in all cases.
-Top level utilities will apply to both telescopes, all instruments, then each level down will have it's own utilities.
+Control Package Utilities are analogous to the utilities discussed in `Observing Utilities`_, but have been evolved and
+moved into the production code areas. Sanctioned Control Utilities will exist at multiple levels.
+These utilities will primarily be called by jobs for the Queue, but not in all cases.
+Top-level utilities will apply to both telescopes, all instruments, then each level down will have it's own utilities.
 An example of this could (not necessarily will) be the centering utility described above, since the desired
 position for stars in LATISS will differ from the main telescope.
 
-Control Utilities all require unit tests, many of which will require data to perform. This will require a central
-repo/place where this data is stored.
-
-.. TODO::
-    DM has developed a way to do this, we should start this discussion. We might need a sample EFD set to go with the
-    data as well.
+Utilities should be as atomic as possible and may not perform actions that get performed by the control classes
+(e.g. ATTCS and LATISS), such as slewing the telescope.
 
 The utilities will live in the `ts_observatory_control` repo with the Control Classes.
 
 
+Required Testing
+----------------
+
+All code in the `ts_observatory_control` requires documentation to a level where other developers can diagnose the
+utility and fix any issues that are resulting in failed tests. This must include a description of the utility, a
+description of the inputs/outputs, and depending on the complexity of the function, examples may be required.
+
+Each utility must come with a set of tests (and accompanying data if required), tests must include:
+
+- Validation of appropriate input types (dtypes)
+
+    - Verification of appropriate input values are only required if the values are not checked/verified elsewhere (such
+      as at lower levels (e.g. the CSCs).
+
+- Testing of end-to-end functionality for the primary functions for appropriate inputs
+
+    - E.g. does it correctly measure the centroid on a piece of test data to within a given tolerance?
+
```

Comment:
Maybe add:

- Testing that common edge cases are properly captured/treated. 

?